### PR TITLE
Remove PHP 7.0 support and test against PHP 7.1 and 7.2

### PR DESCRIPTION
**Pull request by Soullivaneuh**

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
Hi @Soullivaneuh,

for what it means we won't support unmaintained versions of PHP, how about only keep PHP 5.6, PHP 7.1 and PHP 7.2?

Mickaël

**Comment by Soullivaneuh** [comment]
@mickaelandrieu Done, I updated the composer.json file accordingly.

BTW, using unbound constraint like `>=5.4` is not recommended. You can't ensure compatibility with PHP8, but older version will still be installable.

**Comment by Soullivaneuh** [comment]
Weird Travis issue: https://travis-ci.org/PrestaShop/TranslationToolsBundle/jobs/309377885#L510

We are not under 5.4 but 5.6... Maybe a Travis bug?

**Comment by mickaelandrieu** [comment]
> We are not under 5.4 but 5.6... Maybe a Travis bug?

or from Composer? really weird

> BTW, using unbound constraint like >=5.4 is not recommended.

Yeah, I'm aware about it... to be honest I rediscover this bundle with you ^^ if you want to update to a safer condition it's ok :+1:

**Comment by Quetzacoalt91** [comment]
Something stored in the composer.lock may break the build?

Because the PHP version displayed in cli mode is correct and confirmed a few lines before.

**Comment by mickaelandrieu** [comment]
would you mind to remove or update the plaform information?

```
"platform": {
       "php": "5.4.0"
     }
```

File: composer.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     }
   },
   "require": {
-    "php": ">=5.4",
+    "php": "^5.6 || ^7.1",
```
Comment (mickaelandrieu) [inline composer.json]: maybe ^5.4 || 7.1 is better, as PrestaShop 1.7.3 will be the very last version to support php 5.4

Comment (Soullivaneuh) [reply composer.json]: Well, in this case we have to rollback tests again 5.4 and 5.5.

Comment (mickaelandrieu) [reply composer.json]: We don't support unmaintained PHP versions 'but' the minimal requirements of PrestaShop 1.7.3 is PHP 5.4. So the users must be able to install it on PHP 5.4 environnements *but* if bugs happens, it's up to them to fix it.

**Comment by Quetzacoalt91** [comment]
Thank you guys

File: composer.json (hunk 1 above)
Comment (Soullivaneuh) [reply composer.json]: > but if bugs happens, it's up to them to fix it.

Depends of where the bug is.

**Pull request merged**